sdk/core/src/main/java/com/klaviyo/core/config/KlaviyoConfig.kt

```diff
@@ -79,10 +79,16 @@ object KlaviyoConfig : Config {
     /**
      * How many retries to allow an API request before permanent failure
      *
-     * Reasoning: Most likely the rate limit should be cleared within 2 retries with exp backoff.
-     * However, I wanted some extra padding for edge cases, since the consequence is lost data.
+     * Reasoning: Most likely the rate limit should be cleared within 2-3 retries with exp backoff.
      */
-    private const val NETWORK_MAX_RETRIES_DEFAULT: Int = 4
+    private const val NETWORK_MAX_RETRIES_DEFAULT: Int = 50
+
+    /**
+     * Maximum interval between retries for the exponential backoff
```

Comment:
nit: add seconds in the docstring

Reply:
wait, or is this in milliseconds

Reply:
A good reason to state it in the doc string

Reply:
Its 3 minutes, in milliseconds

Reply:
```suggestion
     * Maximum interval between retries for the exponential backoff, in milliseconds
```